checks/normandy/reported_recipes.py

```diff
@@ -29,7 +29,11 @@
 
 
 async def run(
-    api_key: str, server: str, min_total_events: int = 1000, lag_margin: int = 600
+    api_key: str,
+    server: str,
+    min_total_events: int = 1000,
+    lag_margin: int = 600,
+    channels: List[str] = [],
```

Comment:
In Rust I'd suggest the type of this be `Option<Vec>`, instead of giving special meaning to the empty list. Do you think that's worth doing here? (ie, `Optional[List[str]]`) ([docs on Optional](https://docs.python.org/3/library/typing.html#typing.Optional))